### DISCUSSION TITLE
How to only enable hotkey for a button when a specified text box is focused

### DISCUSSION THREAD

**Discussion by xky0007** [AvaloniaUI/Avalonia#16045] [Q&A] [answered]
When a specified text box is focused and edited, a hot key can trigger another button click and call the command. Hot key won't work if not focus on the text box.

Thanks.

**Comment by timunie** [comment, marked as answer]
I'd use a KeyBinding on the TextBox instead. See: 
https://github.com/AvaloniaUI/Avalonia.Samples/blob/d324bcec0d700beabac58b4fb596d0a4478f4b26/src/Avalonia.Samples/CompleteApps/SimpleToDoList/Views/MainWindow.axaml#L64-L66

**Reply by xky0007** [reply]
It works. Thanks. Maybe later we can add `KeyBindings` property inside Textbox. Then we can set it easily.
